internal/proxy/task_scheduler.go

```diff
@@ -170,14 +171,24 @@ func (queue *baseTaskQueue) Enqueue(t task) error {
 		return err
 	}
 
-	ts, err := queue.tsoAllocatorIns.AllocOne(t.TraceCtx())
-	if err != nil {
-		return err
+	var ts Timestamp
+	var id UniqueID
+	if t.CanSkipAllocTimestamp() {
+		ts = tsoutil.ComposeTS(time.Now().UnixMilli(), 0)
```

Comment:
does that mean we need a unique messageID?

If that's the case why don't we just use a local tso instead of using global ids?

Reply:
yes, we need a unique messageID, so we cache a large number of global ids here. If you use local tso, duplicate ids may appear.